fmt/src/formatter.rs

```diff
@@ -920,11 +958,6 @@ impl<'a, W: Write> Visitor for Formatter<'a, W> {
     }
 
     fn visit_event(&mut self, event: &mut EventDefinition) -> VResult {
-        if !event.doc.is_empty() {
-            event.doc.visit(self)?;
-            writeln!(self)?;
```

Comment:
why is it OK to remove all these?

Reply:
The doc comments are no longer attached to individual elements and exist as their own nodes

Reply:
We made that change because solc accepts doc comments anywhere in a file. This is valid solidity:

```
contract c {}
/** foo */
```